she/promote.py

```
@@ -182,6 +182,12 @@ def plan_promotion(
             },
         )
 
+    if verification is not None:
+        if verification.incident_id != incident.incident_id:
+            raise PromotionError("verification incident_id must match incident")
```

Ruff (0.16.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)